tools/pharmcat/pharmcat.xml

```diff
@@ -0,0 +1,80 @@
+<tool id="pharmcat" name="pharmcat" version="@WRAPPER_VERSION@+@VERSION_SUFFIX@" profile="20.01">
+    <macros>
+        <token name="@WRAPPER_VERSION@">1.3.1</token>
+        <token name="@VERSION_SUFFIX@">galaxy0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@WRAPPER_VERSION@">pharmcat</requirement>
```

Comment:
Since there is no conda package the container can not be built. So we need something like that: https://github.com/bgruening/galaxytools/blob/f69dd59785c9674ebd58506d5d8b31289d7e9532/tools/bionano/macros.xml#L26